### PR TITLE
FUSETOOLS-2776 - UI Tests - New "quick" test for "New Fuse Ignite Extension Project" wizard

### PR DESCRIPTION
**Pull request by tsedmik**
https://issues.jboss.org/browse/FUSETOOLS-2776

Signed-off-by: Tomáš Sedmík <tsedmik@redhat.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [x] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [x] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [x] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [x] Have you used the correct file header copyright comment?
- [x] Have you used the correct year in the headers of new files?

### CODE REVIEW

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/condition/ButtonIsEnabled.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import org.eclipse.reddeer.common.condition.AbstractWaitCondition;
+import org.eclipse.reddeer.core.reference.ReferencedComposite;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+
+/**
+ * Wait condition which try whether a push button is enabled
+ * 
+ * @author tsedmik
+ */
+public class ButtonIsEnabled extends AbstractWaitCondition {
```
[the remark this answers is not in the document]
Comment (tsedmik) [reply uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/condition/ButtonIsEnabled.java]: Great ... I was looking for something like this (some already implemented solution)

**Comment by tsedmik** [comment]
@apodhrad - ready for review

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/wizard/NewFuseIgniteExtensionProjectDependenciesPage.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+ * @author tsedmik
+ */
+public class NewFuseIgniteExtensionProjectDependenciesPage extends WizardPage {
+
+	public NewFuseIgniteExtensionProjectDependenciesPage(ReferencedComposite referencedComposite) {
+		super(referencedComposite);
+	}
+
+	public LabeledCombo getSpringBootVersionCMB() {
+		return new LabeledCombo("Spring Boot Version");
```
Comment (apodhrad) [inline uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/wizard/NewFuseIgniteExtensionProjectDependenciesPage.java]: Please reuse the referenced composite, i.e.
```
new LabeledCombo(this, "Spring Boot Version");
```
Also apply the above rule wherever you can.

Comment (tsedmik) [reply uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/wizard/NewFuseIgniteExtensionProjectDependenciesPage.java]: done ;-)

**Comment by apodhrad** [comment]
The test doesn't work from cmd due to missing syndesis deps (fixed in https://github.com/tsedmik/jbosstools-fuse/pull/1).

**Comment by tsedmik** [comment]
@apodhrad ready for review

**Pull request merged**